.github/workflows/pr-verify.yaml

```diff
@@ -12,8 +12,20 @@ jobs:
     runs-on: ubuntu-latest
     name: verify PR contents
     steps:
-    - name: Verifier action
-      id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4 # tag=v0.4.3
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Verifier action
```

Comment:
Please revert the changes to this file (see https://github.com/kubernetes-sigs/cluster-api/pull/10261/files#r1522991772)